woocommerce/woocommerce-ios
# [Manual Taxes M2] Add custom class to transformables in a new model version

Closes: #10639 


## Description

With this PR we add `[String]` as "custom class" in the `cities` and `postcodes` properties of `TaxRate` model. We create a new model version (97) and adapt the code to this new requirement.

## Testing instructions

Ensure that unit tests pass correctly. You can also make sure that the model migrates correctly by running a version with model 96 (trunk) and then installing this version over. You can also test that these properties as properly passed. Before all the array components were passed as a simple element of the array e.g. ["Miami, New York, Los Angeles"], now they should create an array with several elements e.g. ["Miami", "New York", "Los Angeles"]. You can verify that by adding a breakpoint in the `NewTaxSelectorViewModel` once we retrieve the entities from the database. E.g. [here](https://github.com/woocommerce/woocommerce-ios/blob/e0c1efa8d44675b350abc4ee504de084fcc4b163/WooCommerce/Classes/ViewRelated/Orders/Order%20Creation/Taxes/NewTaxRateSelectorViewModel.swift#L128)

## Screenshots

N/A


---
